transport/postis.js

```diff
@@ -37,18 +38,26 @@ function Postis(options) {
   var readynessCheck;
 
   var listener = function(event) {
-    var data;
-    try {
-      data = JSON.parse(event.data);
-    } catch (e) {
+    // Don't try to deserialize messages not sent by matching positis instance
+    if (!event.data || !event.data.postis || event.data.scope !== scope) {
```

Comment:
add log debug message?

Reply:
see my other comment above - this is expected and doesn't make sense to log anything here (would be too often)